.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -397,13 +395,6 @@ jobs:
           cmake --build . --config Release --parallel 4
           ./hello
 
-      - name: Build Sample [simple]
-        run: |
-          ./build.sh -p host-interp
-          python3 ./sample_test_run.py $(pwd)/out
-          exit $?
-        working-directory: ./samples/simple
```

Comment:
Maybe we can update the script later to build simple from wamr-app-framework.

Reply:
I agree.